<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-10T16:29:47 IST" name="Default suite" finished-at="2025-03-10T16:30:33 IST" duration-ms="46139">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.DemoNG@3d285d7e]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2025-03-10T16:29:47 IST" name="Default test" finished-at="2025-03-10T16:30:33 IST" duration-ms="46139">
      <class name="TestRunner.DemoNG">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:TestRunner.DemoNG@3d285d7e]" started-at="2025-03-10T16:29:47 IST" name="setUpClass" finished-at="2025-03-10T16:29:47 IST" duration-ms="650" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@769bd849]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.DemoNG@3d285d7e]" started-at="2025-03-10T16:29:47 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2025-03-10T16:30:33 IST" duration-ms="45399" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["To check the login functionality"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Unknown"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Aniket Singh] but found [test user benakba 1fakhreddine]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Aniket Singh] but found [test user benakba 1fakhreddine]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at Pages.Page2.getUser(Page2.java:25)
at StepDefnition.Sample2.user_verifies_the_username(Sample2.java:30)
at âœ½.user verifies the username(file:///C:/Users/SAI%20%20KATIKI/eclipse-workspace/PageFactoryProject/src/test/resources/Feature/sample3.feature:8)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:TestRunner.DemoNG@3d285d7e]" started-at="2025-03-10T16:30:33 IST" name="tearDownClass" finished-at="2025-03-10T16:30:33 IST" duration-ms="32" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- TestRunner.DemoNG -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
